app/controllers/my_facilities_controller.rb

```diff
@@ -96,7 +93,11 @@ def set_time_zone
     Time.use_zone(time_zone) { yield }
   end
 
-  def authorize_my_facilities
-    authorize(:dashboard, :view_my_facilities?)
+  def authorize_view
+    raise Pundit::NotAuthorizedError unless current_admin.can_read_aggregates?
```

Comment:
There is some scoping missing here, right? Asking if an admin can read something without a particular target doesn't make sense.  Maybe something like:

```
current_admin.accessible_facilities(:view).any?
```

Reply:
We could do that, but `can_read_aggregates?` is a really one-sided relationship at the moment. It really just means [this](https://github.com/simpledotorg/simple-server/pull/1160/files#diff-dbee6fa62dba5eb76fea1e205e2a5f54R57) – basically if you're a super_admin? or you have _any_ access at all. Even if you have the lowest most restrictive Access in the system, as long as you have something, that means you can read aggregates (as it stands).

This definition could change later and we can upgrade `can_read_aggregates?` then?